.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,39 @@
+name: PyPI release
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-and-publish:
+    name: Build and publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: install poetry
+        run: pip install poetry
+      - name: install dependencies
+        run: poetry install
+      - name: install go
+        run: |
+          sudo apt update
+          sudo apt install golang-1.14-go -y
+      - name: build helm bindings
+        run: |
+          bash $(poetry env info -p)/lib/python3.8/site-packages/kapitan/inputs/helm/build.sh
+          bash $(poetry env info -p)/lib/python3.8/site-packages/kapitan/dependency_manager/helm/build.sh
+          mkdir -p commodore/helm_bindings/inputs
+          mkdir -p commodore/helm_bindings/dependency_manager
+          cp $(poetry env info -p)/lib/python3.8/site-packages/kapitan/inputs/helm/{helm_binding.py,libtemplate.so} commodore/helm_bindings/inputs
+          cp $(poetry env info -p)/lib/python3.8/site-packages/kapitan/dependency_manager/helm/{helm_fetch_binding.py,helm_fetch.so} commodore/helm_bindings/dependency_manager
```

Comment:
This will not produce a usable package for users on MacOS.

Reply:
They just cannot use the pre-compiled helm-bindings. Compiling them after the `pip install` still works.

Reply:
But compiling will fail with newer version of go, because kapitan did not release this fix yet: https://github.com/kapicorp/kapitan/commit/a996ea7cda9a1ea863c8462c98e8c9cad8a11480

Reply:
The next version of Kapitan won't provide the Helm bindings at all anymore, and will instead call out to the Helm binary.